edb/pgsql/resolver/sql_introspection.py

```diff
@@ -914,6 +921,17 @@
         ("installed_version", "text"),
         ("comment", "text"),
     ],
+    "pg_backend_memory_contexts": [
```

Comment:
Is it just this table or is there any other?

Reply:
There're at least these:

* `pg_inherits.inhdetachpending`
* `pg_locks.waitstart`
* `pg_prepared_statements.generic_plans`
* `pg_prepared_statements.custom_plans`
* `pg_proc.prosqlbody`
* `pg_replication_slots.two_phase`

I think there's more after `pg_replication_slots`.

Reply:
I've booted up a Postgres 13 instace in docker and generated our introspection catalog from that. It's on branch `sql-pg-13-introspection`.

The problem now is that async-pg is failing because it is expecting `pg_range.rngmultitypid`. Ugh.

Reply:
Does that mean `asyncpg` is also failing on vanilla Postgres 13?

Reply:
A, wait I got it working by generating from 13, and then adding `pg_range.rngmultitypid` (that was added in 14) and removing `pg_ssl.compression` (that was removed in 14).

Reply:
> Does that mean asyncpg is also failing on vanilla Postgres 13?

That I don't know

Reply:
No, asyncpg has a server version switch.  BTW, we should probably spoof Postgres version and fix it at the level of the current pg_catalog support.